src/components/Builder/Achievement/index.jsx

```diff
@@ -270,8 +298,26 @@ const Achievement = ({ achievementData }) => {
           />
         </SortableContext>
       </DndContext>
+      <Modal
```

Comment:
move this confirmation modal in common components and use

Reply:
Done!